brainstormforce/ultimate-addons-for-gutenberg
# Icon list typo issue

### Description

Typography not working in the Icon list.

### Types of changes




Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Created a icon list in v1.14.8
- Checkout to icon-list-typo-issue branch
- Check whether typography working in front-end or not.

### Checklist:
